src/Core/CoreKernel.php

```diff
@@ -357,28 +357,47 @@ protected function getDatabaseConfig()
     }
 
     /**
+     * @return string
+     */
+    protected function getDatabasePrefix()
+    {
+        $prefix = getenv('SS_DATABASE_PREFIX');
+
+        if ($prefix === false) {
+            return '';
```

Comment:
It seems like you are treating `null` as the result for a missing env var. Check http://php.net/manual/en/function.getenv.php: false means not declared, so should be default value here.